docs/modules/ROOT/partials/getting-started/proc-setting-up-kafka-streams-storage.adoc

```diff
@@ -79,6 +79,8 @@ spec:
 ----
 http://example-apicurioregistry-kafkasql.my-project.my-domain-name.com/
 ----
++
+WARNING: The Kafka topic used by {registry} (named `kafkasql-journal` by default) must be configured with a *compaction* cleanup policy, otherwise a data loss may occur.
```

Comment:
```suggestion
WARNING: You must configure the Kafka topic used by {registry} (named `kafkasql-journal` by default) with a compaction cleanup policy, otherwise a data loss may occur.
```

Reply:
Also suggest showing exactly how to do this in the example CRD above: 
```
spec:
  ...
  config:
    cleanup.policy: compact
```

Reply:
Updated!